tests/scripts/deploy_admission_controller.sh

```diff
@@ -51,9 +51,13 @@ echo "$BASE_DIR"
 
 echo "Deploying cert-manager"
 kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/$CERT_VERSION/cert-manager.yaml
+# shellcheck disable=SC2016 # Using single quotes for the command
 timeout 150 sh -c 'until [ $(kubectl -n cert-manager get pods --field-selector=status.phase=Running|grep -c ^cert-) -eq 3 ]; do sleep 1 && echo "waiting for cert-manager pods to be in running state"; done'
-timeout 20 sh -c 'until [ $(kubectl -n cert-manager get pods -o custom-columns=READY:status.containerStatuses[*].ready | grep -c true) -eq 3 ]; do sleep 1 && echo "waiting for the pods to be in ready state"; done'
+# shellcheck disable=SC2016
+timeout 20 sh -c 'ntil [ $(kubectl -n cert-manager get pods -o custom-columns=READY:status.containerStatuses[*].ready | grep -c true) -eq 3 ]; do sleep 1 && echo "waiting for the pods to be in ready state"; done'
+# shellcheck disable=SC2016,SC1083
```

Comment:
Consider SC1083 more here. I think there are actually problems in this code. The single quote before `{.webhooks[*].clientConfig.caBundle}` ends the string and then allows bash to interpret `*` as a glob.

Can we use `jsonpath="{.webhooks[*].clentConfig.caBundle}"` (with double quotes) instead? Or does the single quote need escaped? 

For these commands, it may be better here to not use single quotes and use double quotes instead with escaped `$` where we want. This would get rid of both shellcheck errors I think.

```bash
timeout 20 sh -c "until [ \$(kubectl ... '{.webhooks[*].clientConfig.caBundle}' | grep -c true) -eq 3 ]; do sleep 1 && echo waiting for stuff; done"
```